### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by SergeyOdnosteblets**

### CODE REVIEW

File: src/__tests__/AllUsers.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import '@testing-library/jest-dom';
+
+import axios from 'axios';
+import { BASE_URL, allFetchUsers } from '../utils/axios';
+
+jest.mock('axios');
+
+it('should return all users', async () => {
+  axios.get.mockResolvedValueOnce([]);
```
Comment (t-shevchuk) [inline src/__tests__/AllUsers.test.js:9]: Add empty string after

File: src/__tests__/AllUsers.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  const result = await allFetchUsers();
```
Comment (t-shevchuk) [inline src/__tests__/AllUsers.test.js:10]: and here

File: src/__tests__/AllUsersId.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+import { AllUsersId } from '../components/AllUsersId';
+
+it('rendering a component AllUsersId', () => {
+  const route = '/';
+
+  render(
+    <MemoryRouter initialEntries={[route]}>
```
Comment (t-shevchuk) [inline src/__tests__/AllUsersId.test.js:12]: Why do you use `MemoryRouter` ?

File: src/__tests__/AllUsersId.test.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      <AllUsersId />
+    </MemoryRouter>,
+  );
+
+  const appElement = screen.getByTestId('allUserId');
+  expect(appElement).toBeVisible()
+});
```
Comment (t-shevchuk) [inline src/__tests__/AllUsersId.test.js:19]: Not enough coverage, add also test to check how the component looks like when data is not an empty array

File: src/__tests__/App.test.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react';
+import App from '../App';
+
+it('render app component', () => {
+  render(<App />);
+  const appElement = screen.getByTestId('app');
+  expect(appElement).toBeInTheDocument();
+  expect(screen.queryByRole('App')).toBeNull();
```
Comment (t-shevchuk) [inline src/__tests__/App.test.js:8]: as I know we don't have such role

File: src/components/AllUsersId.jsx [hunk 6]
```diff
@@ -13,6 +13,8 @@ export const AllUsersId = () => {
   }, []);
 
   return (
-    <div className="App">{usersId && usersId.map((item) => <List id={item} key={item.id} />)}</div>
+    <div className="App" data-testid="allUserId">
```
Comment (t-shevchuk) [inline src/components/AllUsersId.jsx:16]: for `data-testid` is better to use kebab case
